### ISSUE TITLE
More information about the UI Preferences to easier configure the DynamoSettings.xml manually

### ISSUE THREAD

**Issue by Thomas84** [DynamoDS/DynamoWishlist#267]
### Dynamo Version

3.03

### Host

Revit 2025.1

### What improvement/feature would you like to see in Dynamo?

Some settings have no obvious reference to the values in the DynamoSettings.xml
It would be nice if would be somewhere a documentation about all the DynamoSettings.XML values and the effect of the values. For example geometry scaling in the UI is small, medium, large and extra large. In the Dynamo Settings it is -2, 0, 2, 4. 
![image](https://github.com/DynamoDS/DynamoWishlist/assets/5624034/8827a7e8-abe2-4b62-8d43-41a13bbe65bd)

### How would you see this improvement/feature working?

A help section in the primer with all settings explained and additionally more information in the preferences:
![image](https://github.com/DynamoDS/DynamoWishlist/assets/5624034/b4cb8fe0-824c-4761-8ed2-4e6749909fc6)

**Comment by Amoursol** [comment]
@Thomas84 is the primary ask for a better understanding of how Preferences map to the actual DynamoSettings.XML file? The -2/0/2/4 doesn't really mean much beyond the ranges they support.

**Comment by Thomas84** [comment]
yes, there is no documentation about how to setup the DynamoSettings.xml properly for deployment. For example the environment variables making problems, and I don't know if I can include settings released for later Dynamo version in older Dynamo Version setting files: https://forum.dynamobim.com/t/custom-dynamo-settings-for-2-19-does-not-load/101640 
It took me a very long time to put everything together. I can provide you our internal documentation (Notion page) as starting point for a primer section, but most of it is documented German. Some settings are explained in https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCore/Configuration/PreferenceSettings.cs but normal users will never find that source. 
Also the DynamoSettings.xml is not alphabetical sorted. And if you have the German UI in Dynamo, it is hard to find the right setting in the DynamoSettings.xml
